persistence/src/main/java/edu/unc/lib/dl/persist/services/edit/ExpireEmbargoService.java

```diff
@@ -65,76 +61,71 @@
     private TransactionManager txManager;
     private SparqlQueryService sparqlQueryService;
 
-    private static final Timer timer = TimerFactory.createTimerForClass(EditTitleService.class);
+    private static final Timer timer = TimerFactory.createTimerForClass(ExpireEmbargoService.class);
 
     public ExpireEmbargoService() {
     }
 
     // run service every day 1 minute after midnight
     @Scheduled(cron = "0 1 0 * * *")
     public void expireEmbargoes() {
-        FedoraTransaction tx = txManager.startTransaction();
+        // get list of expired embargoes
+        List<String> resourceList = getEmbargoInfo();
+        Collection<PID> pids = new ArrayList<>();
 
-        try (Timer.Context context = timer.time()) {
-            // get list of expired embargoes
-            List<String> resourceList = getEmbargoInfo();
-            Collection<PID> pids = new ArrayList<>();
+        // remove all expired embargoes
+        for (String rescUri: resourceList) {
+            FedoraTransaction tx = txManager.startTransaction();
 
-            // remove all expired embargoes
-            for (int i = 0; i < resourceList.size(); i++) {
-                String rescUri = resourceList.get(i);
+            try (Timer.Context context = timer.time()) {
                 PID pid = PIDs.get(rescUri);
                 RepositoryObject repoObj = repoObjLoader.getRepositoryObject(pid);
-                Model model = ModelFactory.createDefaultModel().add(repoObj.getModel());
-                Resource resc = model.getResource(rescUri);
+                Resource resc = repoObj.getResource();
 
                 String eventText = null;
                 boolean expiredEmbargo = false;
-                String embargoDate = resc.getProperty(embargoUntil).getString();
 
-                if (resc.hasProperty(embargoUntil)) {
-                    repoObjFactory.deleteProperty(repoObj, embargoUntil);
-                    pids.add(pid);
-                    expiredEmbargo = true;
-                }
+                // remove embargo
+                String embargoDate = resc.getProperty(embargoUntil).getString();
+                repoObjFactory.deleteProperty(repoObj, embargoUntil);
+                pids.add(pid);
+                expiredEmbargo = true;
 
                 if (expiredEmbargo) {
```

Comment:
I think this is always going to be true now, so you can remove the variable and this conditional